app/MMB/src/mmbList/ImageBadges.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import * as React from 'react';
+import { AdaptableBadge, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type Props = {|
+  status: string,
+  bookingId: string,
+|};
+
+export default function ImageBadges(props: Props) {
+  let statusBadge = null;
+  switch (props.status) {
+    case 'REFUNDED':
+      statusBadge = (
+        <AdaptableBadge
+          translation={<Translation id="mmb.booking_state.refunded" />}
+          style={styles.refundedBadge}
+          textStyle={styles.badgeText}
+        />
+      );
+      break;
+    case 'CONFIRMED':
+      statusBadge = (
+        <AdaptableBadge
+          translation={<Translation id="mmb.booking_state.confirmed" />}
+          style={styles.confirmBadge}
+          textStyle={styles.badgeText}
+        />
+      );
+      break;
+    case 'CLOSED':
+      statusBadge = (
+        <AdaptableBadge
+          translation={<Translation id="mmb.booking_state.in_process" />}
+          style={styles.inProcessBadge}
+          textStyle={styles.badgeText}
+        />
+      );
+      break;
+    default:
+      break;
+  }
+  return (
+    <React.Fragment>
+      {statusBadge}
+      <AdaptableBadge
+        text={props.bookingId}
+        style={styles.bookingIdBadge}
+        textStyle={styles.badgeText}
+      />
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  confirmBadge: {
+    backgroundColor: 'rgba(51, 189, 27, 0.5)',
```

Comment:
Badges should be partially transparent (with blur but we can skip the blur for now)...

Reply:
@tbergq What about the opacity? Not now? :)

Reply:
Oh I see - you used opacity but it didn't work? Or it was not so obvious? 🤔

Reply:
it looks different using opacity: 0.5, don't know why 🤷‍♂️

Reply:
> we can skip the blur for now
Also I though you meant to remove it

Reply:
Ok, I'll rephrase: there should be an opacity (I didn't see it) but let's skip the blur effect. Sorry... 🙈 (https://zpl.io/2Gmv4n4)

Reply:
Ok, I will check it on monday